xwiki-platform-core/xwiki-platform-help/xwiki-platform-help-ui/src/main/resources/Help/WebHome.xml

```diff
@@ -20,24 +19,246 @@
  * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
 -->
 
-<xwikidoc version="1.2" reference="Help.WebHome" locale="">
+<xwikidoc version="1.3" reference="Help.WebHome" locale="">
   <web>Help</web>
   <name>WebHome</name>
   <language/>
   <defaultLanguage/>
   <translation>0</translation>
   <creator>xwiki:XWiki.Admin</creator>
-  <creationDate>1459334154000</creationDate>
+  <creationDate>1459352154000</creationDate>
   <parent>Main.WebHome</parent>
   <author>xwiki:XWiki.Admin</author>
   <contentAuthor>xwiki:XWiki.Admin</contentAuthor>
-  <date>1459334154000</date>
-  <contentUpdateDate>1459334154000</contentUpdateDate>
+  <date>1512001811000</date>
+  <contentUpdateDate>1512001811000</contentUpdateDate>
   <version>1.1</version>
   <title/>
   <comment/>
   <minorEdit>false</minorEdit>
   <syntaxId>xwiki/2.1</syntaxId>
-  <hidden>true</hidden>
-  <content/>
+  <hidden>false</hidden>
+  <content>{{include reference="Help.Code.VelocityMacros" /}}
```

Comment:
This does not seems to be the same content att all. Looks like you exported an older version or something like this.

Reply:
Okay I’ll take a look